modules/470-chrony/templates/daemonset.yaml

```diff
@@ -284,6 +358,59 @@ spec:
   {{- if not (.Values.global.enabledModules | has "vertical-pod-autoscaler") }}
             {{- include "chrony_resources" . | nindent 12 }}
   {{- end }}
+      - image: {{ include "helm_lib_module_image" (list . "chronyExporter") }}
+        name: chrony-exporter
+        {{- include "helm_lib_module_container_security_context_privileged" . | nindent 8 }}
```

Comment:
We must evaluate whether privileged mode is strictly necessary.
For me, a simple root should be enough, moreover, we can try to avoid setting the root user for this container as well.

The only reason for running this exporter as root is that it has to have access to the chrony's Unix socket file, right?
If this statement is correct, we can try to set `fsGroup: 64535` for the pod. In this case, chrony's Unix socket file should inherit the group id and should be accessible from the unprivileged exporter container, thus we can use `helm_lib_module_container_security_context_read_only_root_filesystem` for it.

Reply:
@lazovskiy You was right, tested it in cluster and everything is ok.
Fixed in last commit.